src/common/config.cc

```diff
@@ -874,7 +874,7 @@ int md_config_t::set_val(const char *key, const char *val, bool meta)
   }
 
   // couldn't find a configuration option with key 'key'
-  return -ENOENT;
+  return -EINVAL;
```

Comment:
@tchaikov, indeed, I didn't find a case for "config set" to return an error as "No such file or directory"... Could you enlighten me? I could handle it as a special case, thanks!